Makefile

```diff
@@ -10,6 +10,9 @@ TESTS = $(wildcard test/*.py)
 serve:
 	export PROJECTPATH=${PROJECTPATH} && gulp prod-js && cd ./src && python3 server.py
```

Comment:
I feel ya :+1: . I'm just working on some backend stuff, and gulp is failing silently like we mentioned on thursday. I don't need gulp or all.js at the moment, and consistent gulp compilation is a problem for the frontend to hash out (have fun with that, lol)